src/test/java/nextstep/courses/domain/SessionTest.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.courses.domain;
+
+import java.time.LocalDateTime;
+
+class SessionTest {
```

Comment:
```suggestion
class SessionTestFixture {
```

클래스명이 내용을 반영하지 못하고 있어요.